src/canvas-extractor/tests/data_generation/data_remover/__main__.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Including the Canvas script work-in-progress here. Referenced in the split ticket.